tests/model_serving/model_server/operator/test_operator.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.deployment import Deployment
+from ocp_resources.exceptions import MissingResourceResError
+
+from utilities.constants import APPLICATIONS_NAMESPACE
+
+
+@pytest.fixture(scope="class")
+def odh_model_controller_deployment(admin_client: DynamicClient):
```

Comment:
please add test steps of the test u are performing

Reply:
updated